src/common/options.h

```diff
@@ -91,7 +91,7 @@ struct Option {
   std::vector<const char*> see_also;
 
   value_t min, max;
-  std::vector<std::string> enum_allowed;
+  std::vector<const char*> enum_allowed;
```

Comment:
Are we sure we want char* rather than std::string here? Not that I'm DEAD SET AGAINST IT, but if we're doing std::string operations with them elsewhere this might be a false economy.

Reply:
yes, for smaller memory foot print. a temporary `std::string()` will be created from `const char*` in `Option::dump()`, and in `Option::validate()`, STL offers `operator==(const char*, const std::string)`.